src/pkg/cli/client/byoc/gcp/byoc.go

```diff
@@ -285,7 +285,10 @@ func (b *ByocGcp) AccountInfo(ctx context.Context) (*client.AccountInfo, error)
 	}
 	email, err := b.driver.GetCurrentAccountEmail(ctx)
 	if err != nil {
-		return nil, err
+		if email, err2 := gcp.GetGcloudAccountEmail(); err2 == nil {
```

Comment:
I think it's a bit ugly to have error messages with new-lines. A better way to handle this is to separate the error from the hint. Create a error type(s) and have the caller cobra command handle the error(s) and print the hint.